src/ai/backend/test/templates/auth/login.py

```diff
@@ -24,8 +24,8 @@ def name(self) -> str:
     async def _context(self) -> AsyncIterator[None]:
         test_id = TestIDContext.current()
         test_id_str = str(test_id)
-        credential = LoginCredentialContext.current()
-        endpoint = EndpointContext.current()
+        credential = LoginCredentialConfigContext.current()
+        endpoint = EndpointConfigContext.current()
```

Comment:
I want to consider cases that are not Config, so I would prefer not to use the name "Config".